### PR TITLE
Update test matrix to latest 3 sets of k8s releases

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #347

### CODE REVIEW

**Comment by ckotzbauer** [comment]
We have to wait for the 1.21.0 image:
https://github.com/kubernetes-sigs/kind/issues/2188

**Comment by evrardjp** [comment]
@dholbach I have fixed the merge conflict, rebased, and fixed your change. Here it comes!

**Review by ckotzbauer — APPROVED** [review]

**Comment by evrardjp** [comment]
I think we'll need a new kind.

File: .github/workflows/on-pr-charts.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ jobs:
 
       - name: Create default kind cluster
         uses: helm/kind-action@v1.1.0
+        with:
+          version: v0.10.0
```
Comment (evrardjp) [inline .github/workflows/on-pr-charts.yaml:40]: can we use main?

Comment (evrardjp) [reply .github/workflows/on-pr-charts.yaml:40]: I will answer to my own question: No.
```curl -sSLo "$kind_dir/kind" "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"```

Comment (evrardjp) [reply .github/workflows/on-pr-charts.yaml:40]: We'll need to wait for the latest release.

I was too impatient.

**Pull request merged**